src/test/shared/clients/codeCatalystClient.test.ts

```diff
@@ -0,0 +1,53 @@
+/*!
```

Comment:
Tests for the CC client already live in `./src/test/caws/codecatalystClient.test.ts`